app/src/main/res/layout/fragment_pocket_video.xml

```diff
@@ -67,7 +67,7 @@
 
     <ImageView
         android:id="@+id/pocketHelpButton"
-        tools:ignore="ContentDescription"
+        android:contentDescription="@string/pocket_home_tutorial_description"
```

Comment:
My understanding is that this is the "?" button in the Pocket fragment. Why is this description helpful? Is it because it brings the tutorial text here, to this button, instead of making the user click through a screen? I don't like providing a different experience to users, e.g. if the onboarding experience changes to be more comprehensive, screen reader users may be missing out if they don't know to click through. I'd rather this button be called "Help" or something similar.

Reply:
Raised my concerns: https://github.com/mozilla-mobile/firefox-tv/issues/1125#issuecomment-438501104